### DISCUSSION TITLE
Syslog verbosity

### DISCUSSION THREAD

**Discussion by germanbravouy** [Ylianst/MeshCentral#5557] [Q&A]
Hi there, i need help clarifying if there is any way to increase verbosity and improve the syslog messages of meshcentral.

I just deployed meshcentral with the key values for config.json as follows:
`"syslog": true,
"syslogcp": "ip:port"`

And im receiving logs of meshcentral in a syslog server succesfully.
Unfortunatelly the syslogs messages seems a little shortened in information compared to the logs that are shown in the web UI of meshcentral.

For example in the web UI i can see:
- user -> computer -> session multiplex started
While in the syslog server i only see:
- Started desktop multiplex session

In this way the logs are not sufficient for auditing tasks.
Is there any way to increase the syslog's verbosity please?
I would like to see the same logs as those shown in the web interface

**Comment by Andromeda175** [comment]
I too wish this functionality.

**Comment by q16marvin** [comment]
does something happen to this question? i also see only few syslogs in my syslogserver and missing lot stuff...

**Reply by si458** [reply]
what sorta stuff are you seeing and what sorta stuff are you missing?
if i remember the `syslog` and `syslogtcp` can work together
